.gitignore

```diff
@@ -19,6 +19,7 @@
 /keys
 /nbproject
 /node_modules
+/tags
```

Comment:
It's a Ctags file, so that text editors like vim can understand syntax of the language and provide features like "go to definition".

I'd like to keep it if it doesn't bother you too much, otherwise I'll just remember to remove "/tags" file every time before pushing the code.